admin/spec/back-end/controllers/retro-input-assistant.spec.js

```diff
@@ -95,9 +104,57 @@ describe('retro input assistant controller:', () => {
       const res = getRes()
       spyOn(res, 'redirect')
       spyOn(retroInputAssistantService, 'save')
+      spyOn(accessArrangementsService, 'getCurrentViewMode').and.returnValue(aaViewModes.edit)
       await sut.postSubmitRetroInputAssistant(req, res, next)
       expect(retroInputAssistantService.save).toHaveBeenCalledTimes(1)
       expect(res.redirect).toHaveBeenCalledWith('/access-arrangements/overview?hl=slug')
     })
   })
+
+  describe('getDeleteRetroInputAssistant route', () => {
+    const reqParams = (urlSlug) => {
+      return {
+        method: 'GET',
+        url: `/access-arrangements/delete-retro-input-assistant/${urlSlug}`,
+        params: {
+          pupilUrlSlug: 'pupilUrlSlug'
+        }
+      }
+    }
+
+    it('redirects to error page if edit mode not available', async () => {
```

Comment:
Ideally and if time allows these existing Jasmine tests in the admin app should be converted to Jest.  New tests should be written for Jest.

Reply:
i will refactor these as part of the ticket